metrics/sample.go

```diff
@@ -10,7 +10,7 @@ import (
 
 const rescaleThreshold = time.Hour
 
-// Samples maintain a statistically-significant selection of values from
+// Sample methods maintain a statistically-significant selection of values from
```

Comment:
Please don't add `methods` all over, it's a simple interface.

Reply:
Thank you - updated.